### PR TITLE
log error for failed open

### PR DESCRIPTION
**Pull request by raghavendra-talur**
Signed-off-by: Raghavendra Talur <rtalur@redhat.com>

### CODE REVIEW

File: apps/glusterfs/app.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func NewApp(configIo io.Reader) *App {
 	// Setup BoltDB database
 	app.db, err = bolt.Open(dbfilename, 0600, &bolt.Options{Timeout: 3 * time.Second})
 	if err != nil {
-		logger.Warning("Unable to open database.  Retrying using read only mode")
+		logger.Warning("Unable to open database, Err: %v. Retrying using read only mode", err)
```
Comment (obnoxxx) [inline apps/glusterfs/app.go]: Generally good, but this `"Unable to open database, Err: %v."` reads a little awkward.
I find that usually s/th like this is sufficient: `"Unable to open database: %v."

For readability of the logs, we might also consider doing a new message / line for the `"Retrying using read only mode"` message.

**Comment by obnoxxx** [comment]
ping @raghavendra-talur

**Comment by raghavendra-talur** [comment]
@obnoxxx I updated the log message for simpler err message display. I did not move the second sentence to new line because it won't get [heketi] header if done that way. Compare the following two logs:

```
[heketi] ERROR 2017/11/06 17:42:03 /src/github.com/heketi/heketi/apps/glusterfs/app.go:100: Unable to open database: transport endpoint is not connected.
Retrying using read only mode 
```

```
[heketi] ERROR 2017/11/06 17:44:29 /src/github.com/heketi/heketi/apps/glusterfs/app.go:100: Unable to open database: transport endpoint is not connected. Retrying using read only mode
```

**Comment by obnoxxx** [comment]
@raghavendra-talur thanks. The multi-line could be achieved by using two log messsages. but that's fine. Why have you now changed `logger.Warning to logger.LogError ?

**Comment by obnoxxx** [comment]
@raghavendra-talur you may have missed this question:

> Why have you now changed `logger.Warning to logger.LogError ?

**Comment by obnoxxx** [comment]
@raghavendra-talur forget it. looked at the code around the message and error certainly seems the appropriate level. ==> merging

**Pull request merged**